### ISSUE TITLE
[BUG] `su` inside a pot does not work due to wrong root fs permissions

### ISSUE THREAD

**Issue by zilti** [bsdpot/pot#249]
**Describe the bug**
Trying to `su` to another user account inside a jail created with Pot will fail due to wrong access permissions for `/`.

**To Reproduce**
1. Create a pot
2. Try to change to another user with `su`
3. See error

**Expected behavior**
I expect to get a shell for that other user

**Additional context**
The workaround is to run `chmod 0755 /` inside the jail.

**Comment by grembo** [comment]
This is a duplicate of #233 

It has already been fixed in master (#234)

@pizzamig what about doing a new release soon (maybe after #240 has been merged).

**Comment by grembo** [comment]
Duplicate of #233
